Source/WTF/wtf/glib/GSpanExtras.h

```diff
@@ -66,6 +66,12 @@ GMallocSpan<T, Malloc> adoptGMallocSpan(std::span<T> span)
 
 WTF_EXPORT_PRIVATE GMallocSpan<char*, GMallocStrv> gKeyFileGetKeys(GKeyFile*, const char* groupName, GUniqueOutPtr<GError>&);
 WTF_EXPORT_PRIVATE GMallocSpan<GParamSpec*> gObjectClassGetProperties(GObjectClass*);
+WTF_EXPORT_PRIVATE GMallocSpan<const char*> gVariantGetStrv(GVariant*);
+
+static inline GMallocSpan<const char*> gVariantGetStrv(GRefPtr<GVariant> variant)
```

Comment:
Not sure you need this overload. May be better to call `.get()` at call sites. If you prefer to keep it, I'd suggest passing a `const GRefPtr<GVariant>&` to avoid bumping the refcount for no reason.

Reply:
I'll remove one of the versions, we only need the one with `GRefPtr`.